client/my-sites/controller.js

```diff
@@ -189,11 +189,16 @@ function isPathAllowedForDomainOnlySite( path, slug, primaryDomain, contextParam
 		emailManagementTitanControlPanelRedirect,
 	];
 
+	// Injects required parameters for some paths as otherwise comparison won't work well
```

Comment:
I am not sure what this comment means. What comparison won't work?

Reply:
Path comparison. I tried to refine the original comment that was located a few lines below but obviously this wasn't enough. I'll clarify that 🙂.